visual-recognition/src/main/java/com/ibm/watson/developer_cloud/visual_recognition/v3/model/SimilarImage.java

```diff
@@ -0,0 +1,38 @@
+package com.ibm.watson.developer_cloud.visual_recognition.v3.model;
+
+import java.util.Date;
+
+import com.google.gson.annotations.SerializedName;
+import com.ibm.watson.developer_cloud.service.model.GenericModel;
+
+public class SimilarImage extends GenericModel {
+	@SerializedName("image_id")
+	private String image_id;

```

Comment:
Use camelCase instead of snake_case for parameters and method.

There are other services with models that you can use as example